ibacm/src/acm.c

```diff
@@ -216,14 +217,14 @@ static struct sa_data {
 static const char *acme = IBACM_BIN_PATH "/ib_acme -A";
 static const char *opts_file = ACM_CONF_DIR "/" ACM_OPTS_FILE;
 static const char *addr_file = ACM_CONF_DIR "/" ACM_ADDR_FILE;
-static char log_file[128] = IBACM_LOG_FILE;
+static char log_file[NAME_MAX] = IBACM_LOG_FILE;
 static int log_level = 0;
-static char lock_file[128] = IBACM_PID_FILE;
+static char lock_file[NAME_MAX] = IBACM_PID_FILE;
```

Comment:
Ditto

Reply:
.. and in acm_set_options() please use strncpy() instead of strcpy when copying into these three variables.